java/apps/authentication-portal/src/main/webapp/smsOtp.jsp

```diff
@@ -109,136 +111,85 @@
                 <jsp:include page="includes/product-title.jsp"/>
                 <% } %>
             </layout:component>
-            <layout:component componentName="MainSection">
```

Comment:
branding-preferences should be imported from /includes. This change is already included in the new-ui branch.

Please check the other files too.